File: Herencia/python/Vehiculo.py
class Vehiculo(object):
    def __init__(self):
        self.llantas = 4
        self.estado = False
        self.puertas = 4
        self.asientos = "Piel"
        self.largo = 1.20

    def encender(self):
        self.estado = True

    def apagar(self):
        self.estado = False

    def getEstado(self):
        return self.estado

    def sonar(self):
        print("pi-pi")

File: Herencia/Java/Vehiculo.java
public class Vehiculo{
    public int llantas;
    public boolean estado;
    public int puertas;
    public String asientos;
    public double largo;

    public Vehiculo(){
        this.llantas = 4;
        this.estado = false;
        this.puertas = 4;
        this.asientos = "Piel";
        this.largo = 1.20;
    }

    public void encender(){
        this.estado = true;
    }

    public void apagar(){
        this.estado = false;
    }

    public boolean getEstado(){
        return this.estado;
    }

    public void sonar(){
        System.out.println("pi-pi");
    }
}